charms/docker/compose.py

```diff
@@ -47,6 +47,19 @@ def kill(self, service=None):
             cmd = "docker-compose kill"
         self.run(cmd)
 
+    def rm(self, service=None):
+        '''
+        Convenience method that wraps `docker-compose rm`
+
+        usage: c.rm('nginx') to remove the 'nginx' service from the
```

Comment:
I worry 'rm' without params might be problematic. Think of the case where a
user has a variable that gets set to None or '' by logic accident. I'm more
on the side of explicit either '*' or a 'rm_all'

On Tue, Dec 8, 2015, 5:04 PM Matt Bruzek <notifications@github.com> wrote:

> In charms/docker/compose.py
> <https://github.com/juju-solutions/charms.docker/pull/9#discussion_r47020999>
> :
>
> > @@ -47,6 +47,19 @@ def kill(self, service=None):
> >              cmd = "docker-compose kill"
> >          self.run(cmd)
> >
> > +    def rm(self, service=None):
> > +        '''
> > +        Convenience method that wraps `docker-compose rm`
> > +
> > +        usage: c.rm('nginx') to remove the 'nginx' service from the
>
> I see that the wording is copied this from above replacing 'run' with
> 'remove'. This comment could be interpreted that the rm edits the
> docker-compose.yml file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju-solutions/charms.docker/pull/9/files#r47020999>.
>